src/csv.ts

```diff
@@ -149,7 +149,11 @@ export function testCaseFromCsvRow(row: CsvRow): TestCase {
     'metadata',
   ].map((k) => `_${k}`);
 
-  for (const [key, value] of Object.entries(row)) {
+  // Remove leading and trailing whitespace from keys, as leading/trailing whitespace interferes with
+  // meta key parsing.
+  const sanitizedRows = Object.entries(row).map(([key, value]) => [key.trim(), value]);
```

Comment:
Added this because an accidental space in my csv resulted in an assert being treated as a variable.